commands/peers/addpeer.go

```diff
@@ -43,6 +49,44 @@ func addPeerHandler(w http.ResponseWriter, r *http.Request) {
 		rest.SendHTTPError(w, http.StatusInternalServerError, *rsp.OpError)
 		return
 	}
+
+	// Add member to etcd server
```

Comment:
We'll need to do a lot of etcd actions in multiple other places. So it will be good if you get the etcd client initialized in a central location, and use that everywhere, instead of creating a new client every time.